src/styles/styles.js

```diff
@@ -798,6 +798,50 @@ const styles = {
         justifyContent: 'center',
     },
 
+    emojiPickerContainer: {
+        backgroundColor: themeColors.componentBG,
+        minWidth: CONST.EMOJI_PICKER_SIZE,
+    },
+
+    emojiPickerList: {
+        height: 300,
+        width: '100%',
+        paddingHorizontal: 16,
+    },
+
+    emojiHeaderStyle: {
+        backgroundColor: themeColors.componentBG,
+        width: '100%',
+        paddingVertical: 12,
+        fontFamily: fontFamily.GTA_BOLD,
+        fontWeight: fontWeightBold,
+        color: themeColors.heading,
+        fontSize: variables.fontSizeSmall,
+    },
+
+    // Emoji Picker Styles
+    emojiText: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.iconSizeLarge,
+        paddingTop: 4,
+        paddingBottom: 4,
+        paddingRight: 8,
+        paddingLeft: 8,
+    },
+
+    emojiItem: {
+        width: '12.5%',
```

Comment:
Why do `12.5%` instead of a constant value ? Is the width of the parent variable?

Reply:
100/8 = 12.5, so if we want columns of 8, that's what we gotta do.

Reply:
I thought because we know the width of the parent and it doesn't change that we might want to just use a constant. But if it works it works. Mainly just curious.